data/free worlds war jobs.txt

```diff
@@ -0,0 +1,722 @@
+# Copyright (c) 2018 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "FW Scout Run [0]"
+	job
+	repeat
+	name `Scouting Run`
+	description `Fly through the designated Republic system and scout it out for Navy activity. Then return to <destination> by <date> where your ship's scanner logs will be analyzed. Payment is <payment>.`
```

Comment:
Could just be `Scout the <waypoints> for Navy activity and return to...`

Reply:
Is `<waypoint>` a text replacement? I don't see it on the wiki.

Reply:
It is not currently. But it should be.

Reply:
Added in #3336 , to include just the bare minimum of the system name and the joining commas and `and`s